content/docs/using-tina-editor.md

```diff
@@ -1,45 +1,43 @@
 ---
 title: Using the Tina Editor
 id: /docs/using-tina-editor/
-last_edited: '2021-12-03T00:00:00.000Z'
+last_edited: '2022-01-09T00:00:00.000Z'
 next: '/docs/schema'
 ---
 
-Tina is very flexible & extendable, so any Tina site may have a unique experience. Here we'll document the out-of-the-box approach that most of our starters use.
+Tina is very flexible & extendable, so any Tina site may have a unique experience. As an introduction, we'll cover the out-of-the-box experience that most of our starters implement.
 
-## Entering Edit-mode
+## Entering the Admin
```

Comment:
A bit confused on why this is "Admin" could it be something more general like "Entering the editing experience"